packages/core/src/connectivity/scp-cf/environment-accessor.ts

```diff
@@ -294,7 +294,7 @@ function selectXsuaaInstanceWithoutJwt(
   if (!arrayHasExactlyOneElement(xsuaaCredentials)) {
     logger.warn(
       `The following XSUAA instances are bound: ${xsuaaCredentials.map(
-        x => '\n\t- {x.xsappname}'
+        x => `\n\t- ${x.xsappname}`
```

Comment:
This is actually a fix. Not sure why openapi detects it.

Reply:
You are right. I saw the warning when doing linting, hence I fixed it in this PR.